compiler/src/iree/compiler/Codegen/Passes.h

```diff
@@ -445,6 +445,10 @@ LogicalResult verifyGPUMatmulTensorCorePipeline(
 void addGPUMatmulTensorCorePassPipeline(OpPassManager &pm,
```

Comment:
Let us put a header comment here and replace MMA Op with WMMA Op

Reply:
good point, done.